middleware/protect.js

```diff
@@ -49,6 +49,12 @@ module.exports = function (keycloak, spec) {
   }
 
   return function protect (request, response, next) {
+    if (request.route.path) {
```

Comment:
let's follow the approach as discussed.
```javascript
module.exports = (action) => {
    return (req, res, next) => {
        req.keyCloakConfig = {
            action
        }
        next();
    }
}
```
create a new method in the middleware KeycloakActions and set action based on the route.